src/app/core/services/route.service.ts

```diff
@@ -72,7 +67,7 @@ export function parameterSelector(key: string, paramsSelector: (state: CoreState
  */
 @Injectable()
 export class RouteService {
-  constructor(private route: ActivatedRoute, private router: Router, private store: Store<CoreState>) {
+  constructor(private route: ActivatedRoute, private router: Router, private store: Store<AppState|CoreState>) {
```

Comment:
Can't this just be AppState?

Reply:
I move HistoryState from AppState to CoreState, so now it can be only CoreState